Source/Core/DolphinWX/Debugger/MemoryView.cpp

```diff
@@ -123,9 +123,9 @@ wxString CMemoryView::ReadMemoryAsString(u32 address) const
     str.reserve(4);
     for (unsigned int i = 0; i < 4; ++i)
     {
-      u8 byte = static_cast<u8>(mem_data >> (24 - i * 8));
-      if (std::isprint(byte))
-        str += static_cast<char>(byte);
+      char byte = static_cast<char>(mem_data >> (24 - i * 8) & 0xFF);
+      if (std::isprint(byte, std::locale::classic()))
+        str += byte;
```

Comment:
I would use a `.` character instead of ` ` as the former is the _de facto_ standard way of showing that a character is non-printable.